doc/syntax.md

```diff
@@ -128,29 +451,67 @@ Currenly only spike output is supported.
 
    `process_spikes()`
 
-   concepts for refractoriness
+   emit_spike(): calling this function in the `update`-block results in firing
+   a spike through the neuron.
+### Concepts for refractoriness
+In order to model refractory and non-refractory states, two variables are
```

Comment:
Replace "refractory" by "refractoriness" and add "the" in front of "non.refractory".